src/service/cli/generate.js

```diff
@@ -0,0 +1,148 @@
+  `Золотое сечение — соотношение двух величин, гармоническая пропорция.`,
+  `Собрать камни бесконечности легко, если вы прирожденный герой.`,
+  `Освоить вёрстку несложно. Возьмите книгу новую книгу и закрепите все упражнения на практике.`,
+  `Бороться с прокрастинацией несложно. Просто действуйте. Маленькими шагами.`,
+  `Программировать не настолько сложно, как об этом говорят.`,
+  `Простые ежедневные упражнения помогут достичь успеха.`,
+  `Это один из лучших рок-музыкантов.`,
+  `Он написал больше 30 хитов.`,
+  `Из под его пера вышло 8 платиновых альбомов.`,
+  `Процессор заслуживает особого внимания. Он обязательно понравится геймерам со стажем.`,
+  `Рок-музыка всегда ассоциировалась с протестами. Так ли это на самом деле?`,
+  `Достичь успеха помогут ежедневные повторения.`,
+  `Помните, небольшое количество ежедневных упражнений лучше, чем один раз, но много.`,
+  `Как начать действовать? Для начала просто соберитесь.`,
+  `Игры и программирование разные вещи. Не стоит идти в программисты, если вам нравится только игры.`,
+  `Альбом стал настоящим открытием года. Мощные гитарные рифы и скоростные соло-партии не дадут заскучать.`,
+];
+const CATEGORIES = [
+  `Деревья`,
+  `За жизнь`,
+  `Без рамки`,
+  `Разное`,
+  `IT`,
+  `Музыка`,
+  `Кино`,
+  `Программирование`,
+  `Железо`,
+];
+
+const getRandomElement = (array) => array[random(array.length - 1)];
+const getRandomeDate = () => {
+  const MONTHS = 3;
+  const DAYS_IN_MONTH = 30;
+  const HOURS_IN_DAY = 24;
+  const MINNUTES_IN_HOUR = 60;
+  const SECONDS_IN_MINNUTES = 60;
+  const MILLISECONDS_IN_SECOND = 1000;
+  const MS_IN_THREE_MONTH = MONTHS * DAYS_IN_MONTH * HOURS_IN_DAY * MINNUTES_IN_HOUR * SECONDS_IN_MINNUTES * MILLISECONDS_IN_SECOND;
+  const TIMESTAMP = now();
+  const RANDOM_TIMESTAMP = random(TIMESTAMP - MS_IN_THREE_MONTH, TIMESTAMP);
+
+  return new Date(RANDOM_TIMESTAMP).toLocaleString();
+};
+
+const shuffle = (someArray) => {
+  for (let i = someArray.length - 1; i > 0; i--) {
+    const randomPosition = Math.floor(Math.random() * i);
+    [someArray[i], someArray[randomPosition]] = [someArray[randomPosition], someArray[i]];
+  }
+
+  return someArray;
+};
+
+const getAnnounceAndFullText = (sentences) => {
+  const MAX_ANNOUNCE_SENTENCES = 5;
+  const announceSentences = random(1, MAX_ANNOUNCE_SENTENCES);
+  const fullTextSentences = random(MAX_ANNOUNCE_SENTENCES, sentences.length - 1);
+  const shuffledSentences = shuffle(sentences);
+  const announce = take(shuffledSentences, announceSentences);
+  const fullText = take(shuffledSentences, fullTextSentences);
+
+  return {announce, fullText};
+};
+
+const getRandomCategories = (categories) => {
+  const randomLength = random(1, categories.length - 1);
+  const shuffledCategories = shuffle(categories);
+  const preparedCategories = take(shuffledCategories, randomLength);
+
+  return preparedCategories;
+};
+
+const getPublications = (count) => (
+  Array(count).fill({}).map(() => {
```

Comment:
```suggestion
  Array(count).map(() => {
```
Лишнее действие `.fill({})`

Reply:
Консоль говорит обратное
```
Array(5).map(() => 1)
(5) [empty × 5]length: 5
```
заменю на
`[...Array(5)].map(() => 1)`